Gems/Camera/Code/Source/EditorCameraComponent.cpp

```diff
@@ -137,6 +142,8 @@ namespace Camera
                 ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation)
                 ->Attribute(AZ::Script::Attributes::Module, "camera")
                 ->Event("ToggleCameraAsActiveView", &EditorCameraViewRequests::ToggleCameraAsActiveView)
+                ->Event("MatchViewport", &EditorCameraViewRequests::MatchViewport)
+                ->Event("IsThisCamera", &EditorCameraViewRequests::IsThisCamera)
```

Comment:
nit: This name would need to be updated too to `IsActiveCamera` (if you go with my suggested change above)